src/ReconcilerInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+
+namespace Webgriffe\SyliusAkeneoPlugin;
+
+
+interface ReconcilerInterface
+{
+    /**
+     * A string used to identify the Akeneo resource managed by this importer (for example: Product, Category, ecc...)
+     */
+    public function getAkeneoEntity(): string;
+
+    /**
+     * Must return the list of Akeneo identifiers of entities managed by this importer modified after the given
+     * date/time.
+     *
+     * @return string[]
+     */
+    public function getIdentifiersModifiedSince(\DateTime $sinceDate): array;
```

Comment:
See https://github.com/webgriffe/SyliusAkeneoPlugin/pull/74/files#diff-633f4ae3247f981f923424a31ea19c649fda784bea38101dfe323cd8cb9735fcR71

```suggestion
    /**
     * Must return the list of Akeneo identifiers of all entities managed by this reconciler.
     *
     * @return string[]
     */
    public function getAllIdentifiers(): array;
```